src/Postmark/PostmarkClient.php

```diff
@@ -1066,6 +1066,73 @@ function editWebhookConfiguration($id, $url = NULL, $httpAuth = NULL, $httpHeade
 
 		return new DynamicResponseModel($this->processRestRequest('PUT', "/webhooks/$id", $body));
 	}
+
+	/**
+	 * Create Suppressions for the specified recipients.
+	 *
+	 * @param string $suppressionChanges Array of SuppressionChangeRequest objects that specify what recipients to suppress.
+	 * @param string $messageStream Message stream where the recipients should be suppressed. If not provided, they will be suppressed on the default transactional stream.
+	 *
+	 * Suppressions will be generated with a Customer Origin and will have a ManualSuppression reason.
+	 * @return DynamicResponseModel
+	 */
+	function createSuppressions($suppressionChanges = array(), $messageStream = NULL) {
+		$body = array();
+		$body["Suppressions"] = $suppressionChanges;
+		
+		if ($messageStream === NULL) {
+			$messageStream = "outbound";
+		}
+		
+		return new DynamicResponseModel($this->processRestRequest('POST', "/message-streams/$messageStream/suppressions", $body));
+	}
+
+	/**
+	 * Reactivate Suppressions for the specified recipients.
+	 *
+	 * @param string $suppressionChanges Array of SuppressionChangeRequest objects that specify what recipients to reactivate.
+	 * @param string $messageStream Message stream where the recipients should be reactivated. If not provided, they will be reactivated on the default transactional stream.
+	 *
+	 * Suppressions will only be reactivated if you have the required authority.
```

Comment:
```suggestion
	 * Only 'Customer' origin manual suppressions and 'Recipient' origin 'HardBounce' suppressions can be reactivated.
```

I think these are the only scenarios where they can reactivate, yeah? Might be helpful to be more explicit about what they can and can't reactivate themselves.